Dalamud/Interface/Internal/PluginCategoryManager.cs

```diff
@@ -0,0 +1,496 @@
+            ["goodmemory"] = new string[] { "UI" },
+            ["harphero"] = new string[] { "jobs" },
+            ["housemate"] = new string[] { "UI" },
+            ["itemsearchplugin"] = new string[] { "inventory" },
+            ["jobbars"] = new string[] { "jobs" },
+            ["jobicons"] = new string[] { "jobs" },
+            ["kapture"] = new string[] { "UI" },
+            ["kingdomheartsplugin"] = new string[] { "UI" },
+            ["macrochain"] = new string[] { "jobs" },
+            ["maplinker"] = new string[] { "UI" },
+            ["marketboardplugin"] = new string[] { "UI" },
+            ["minicactpotsolver"] = new string[] { "minigames" },
+            ["moaction"] = new string[] { "jobs" },
+            ["namingway"] = new string[] { "jobs", "UI" },
+            ["neatnoter"] = new string[] { "UI" },
+            ["nosoliciting"] = new string[] { "UI", "social" },
+            ["oopsalllalafells"] = new string[] { "other" },
+            ["orchestrion"] = new string[] { "sound" },
+            ["owofy"] = new string[] { "social" },
+            ["peepingtom"] = new string[] { "UI" },
+            ["pennypincher"] = new string[] { "UI", "inventory" },
+            ["pingplugin"] = new string[] { "UI" },
+            ["pixelperfect"] = new string[] { "UI" },
+            ["playertrack"] = new string[] { "UI", "social" },
+            ["prefpro"] = new string[] { "UI" },
+            ["pricecheck"] = new string[] { "UI", "inventory" },
+            ["qolbar"] = new string[] { "UI" },
+            ["quest map"] = new string[] { "UI" },
+            ["remindme"] = new string[] { "UI" },
+            ["rezpls"] = new string[] { "UI" },
+            ["sillychat"] = new string[] { "other" },
+            ["simpletweaksplugin"] = new string[] { "UI" },
+            ["skillswap"] = new string[] { "jobs" },
+            ["slidecast"] = new string[] { "jobs" },
+            ["sonarplugin"] = new string[] { "UI" },
+            ["soundfilter"] = new string[] { "sound" },
+            ["soundsetter"] = new string[] { "sound" },
+            ["teleporterplugin"] = new string[] { "UI" },
+            ["textboxstyler"] = new string[] { "UI" },
+            ["texttotalk"] = new string[] { "UI" },
+            ["thegreatseparator"] = new string[] { "UI" },
+            ["titleedit"] = new string[] { "UI" },
+            ["tourist"] = new string[] { "UI" },
+            ["triadbuddy"] = new string[] { "minigames" },
+            ["vfxeditor"] = new string[] { "UI" },
+            ["visibility"] = new string[] { "UI" },
+            ["voidlist"] = new string[] { "UI" },
+            ["waymarkpresetplugin"] = new string[] { "UI" },
+            ["wintitle"] = new string[] { "UI" },
+            ["woldo"] = new string[] { "UI" },
+            ["wondroustailssolver"] = new string[] { "minigames" },
+            ["xivchat"] = new string[] { "social" },
+            ["xivcombo"] = new string[] { "jobs" },
+        };
```

Comment:
I kinda want to avoid hardcoding this in Dalamud, at least in here - would be better to do it when generating the pluginmaster additionally to having a "uncategorized"

Reply:
Not sure if casing matters but there's "other" and "Other".

Reply:
right, forgot that pluginmaster is also a script that can be messed with. 

can move fallback category generation in there (it's supposed to be temporary anyway, just to avoid forcing people to update manifests right now)